docs/preview/features/powershell/datafactory.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Scripts related to interacting with Azure Data Factory"
+layout: default
+---
+
+# Azure Data Factory
+
+## Installation
+
+To have access to the following features, you have to import the module:
+
+```powershell
+PS> Import-Module -Name Arcus.Scripting.DataFactory
+```
+
+## Start the trigger of a Azure Data Factory
+
+Start a Data Factory V2 Trigger.
+
+| Parameter                   | Mandatory | Description																			     |
```

Comment:
Tabs or spaces?

🏃 💨

Reply:
Hah, didn't notice that!
I'm a tabs-kind of guy myself, so I's go for tabs obviously :)

Reply:
Since spaces is I think mostly recommanded and used everywhere in Arcus, let's go with that.